trust/extract-edk2.c

```diff
@@ -34,11 +34,167 @@
 
 #include "config.h"
 
+#include "buffer.h"  /* p11_buffer */
+#include "debug.h"   /* return_val_if_fail() */
+#include "message.h" /* p11_message() */
 #include "extract.h" /* p11_extract_edk2_cacerts() */
 
+#include <stdint.h>  /* UINT32_MAX */
+#include <limits.h>  /* SSIZE_MAX */
+
+/* types from the UEFI 2.7 spec, section "31.4.1 Signature Database" */
+#pragma pack (1)
```

Comment:
I that pragma supported by gcc or clang? Isn't `__attribute__((packed))` a better alternative for the compilers we use?